Makefile

```diff
@@ -552,6 +552,9 @@ $(eval $(call build-package,ratelimit,0.0_git20230331-r0))
 $(eval $(call build-package,logstash,8.7.0-r0))
 $(eval $(call build-package,ntpd-rs,0.2.1-r0))
 $(eval $(call build-package,trust-dns,0.22.0-r0))
+$(eval $(call build-package,go-md2man,2.0.2-r0))
+$(eval $(call build-package,runc,1.1.5-r0))
+$(eval $(call build-package,lzo,2.10-r0))
```

Comment:
These will need to be updates to packages.txt 

Sorry for the breaking change to the build process, we hope that over time there will be no changes necessary at all except adding new YAMLs.